lib/p2p/Peer.ts

```diff
@@ -839,6 +839,12 @@ class Peer extends EventEmitter {
   }
 
   private handleDisconnecting = (packet: packets.DisconnectingPacket): void  => {
+    if (!this.sessionInitPacket && packet.body && packet.body.reason === DisconnectionReason.Banned) {
```

Comment:
The reason is if we don't have the init packet, that means we aren't connected to the node and as @kilrau stated [here](https://github.com/ExchangeUnion/xud/issues/693), we should only mark the node as banning us if we don't get the `init packet` on the connection attempt.

That is what I understood from the issue.

Reply:
I think that issue is slightly outdated since we aren't using `Hello` packets any more. In any case, in any case I think we will actually get the session init packet even when we are banned, since that is where we communicate node pub keys to peers, and until they know our pub key they won't know if they've banned us. So I'm pretty sure we don't want this check here, unless there's something else I'm missing here.

Reply:
@sangaman is right, the issue description is outdated